cloud/shared/variable_definitions.json

```diff
@@ -41,6 +41,12 @@
     "tfvar": true,
     "type": "string"
   },
+  "ADFS_ADMIN_GROUP": {
```

Comment:
Was this variable lost in some previous PR?

Reply:
it was in both aws and azure configs, and currently that's not allowed. I'm moving it to shared config.
cloud/shared/validate_variable_definitions_test.py is checking for that and it wasn't running before this change.